internal/config/packages.go

```diff
@@ -1,9 +1,15 @@
 package config
 
+import "github.com/spf13/viper"
+
 type Packages struct {
 	Cataloger catalogerOptions `yaml:"cataloger" json:"cataloger" mapstructure:"cataloger"`
 }
 
-func (cfg *Packages) build() error {
-	return cfg.Cataloger.build()
+func (cfg Packages) loadDefaultConfig(v *viper.Viper) {
+	v.SetDefault("package.cataloger.enabled", true)
```

Comment:
these constants would only be used in one spot (this one) ... we wouldn't be able to parameterize the other place where parts of this constant are used (e.g. the struct tag listing `package` to an individual field). I'm quite open to ways that make this safer, it since they are tied to struct tags it would be easy to go to the effort to add a const that has the same problem (someone makes an update elsewhere and it still breaks accidentally).

The point of the change in this PR was to at least bring the setting of the default values into the same file where the values reside (in the struct definition)... it was getting easier to accidentally forget to update the global function where they were all set.